app/views/miq_ae_customization/_dialog_field_form_dynamic_options.html.haml

```diff
@@ -0,0 +1,40 @@
+.form-group
+  %label.col-md-2.control-label
+    = _('Entry Point (NS/Cls/Inst)')
+  .col-md-8
+    = text_field_tag('field_entry_point',
+                      @edit[:field_entry_point],
+                      :class => "form-control",
+                      :onFocus => 'miqShowAE_Tree("field_entry_point");')
+.form-group
+  %label.col-md-2.control-label
+    = _('Show Refresh Button')
+  .col-md-8
+    = check_box_tag('field_show_refresh_button', '1',
+                    @edit[:field_show_refresh_button],
+                    'data-miq_observe_checkbox' => {:url => url}.to_json)
+- if @edit[:field_show_refresh_button] # show only if field_show_refresh_button is on
+  .form-group
+    %label.col-md-2.control-label
+      = _('Load Values on Init')
+    .col-md-8
+      = check_box_tag('field_load_on_init', '1',
+                      @edit[:field_load_on_init],
+                      'data-miq_observe_checkbox' => {:url => url}.to_json)
+
+.form-group
+  %label.col-md-2.control-label
+    = _('Auto refresh')
+  .col-md-8
+    = check_box_tag('field_auto_refresh', '1',
+                    @edit[:field_auto_refresh],
+                    'data-miq_observe_checkbox' => {:url => url}.to_json)
+
+.form-group
+  %label.col-md-2.control-label
+    = _('Auto Refresh other fields when modified')
+  .col-md-8
+    = check_box_tag('field_trigger_auto_refresh', '1',
+                    @edit[:field_trigger_auto_refresh],
+                    'data-miq_observe_checkbox' => {:url => url}.to_json)
```

Comment:
I don't know why I didn't do this when I originally made this form group, but this field could probably be available for all types of dialog fields, right @gmcculloug? Could probably be taken out of this partial and put back into the main one or a separate "common" one (though I don't think there's anything else in common between the dynamic and non-dynamic ones).